modules/serverless-config-replicas.adoc

```diff
@@ -0,0 +1,49 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-HA.adoc
+
+[id="serverless-config-replicas_{context}"]
+= Configuring high availability replicas on {ServerlessProductName}
+
+HA functionality is available by default on {ServerlessProductName} for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, and `kourier-gateway` components. These components are configured with two replicas by default.
+
+You modify the number of replicas that are created per controller by changing the configuration of `KnativeServing.spec.highAvailability` in the KnativeServing custom resource definition.
```

Comment:
Why does the user want to change the number of replicas when HA is configured by default? Extra HA??

Reply:
They can either disable HA completely or they can add more replicas if they feel they need them - I think this in context of Serverless is probably clear enough